run.sh

```diff
@@ -6,13 +6,13 @@ if [[ -f .joplinrc ]];then
 fi
 
 APP_BASE_URL="${APP_BASE_URL:-https://joplin.me.org}"
-APP_PORT="${APP_BASE_URL:-22300}"
-DB_CLIENT="${APP_BASE_URL:-pg}"
-POSTGRES_PASSWORD="${APP_BASE_URL:-joplin}"
-POSTGRES_DATABASE="${APP_BASE_URL:-joplin}"
-POSTGRES_USER="${APP_BASE_URL:-joplin}"
-POSTGRES_PORT="${APP_BASE_URL:-5432}"
-POSTGRES_HOST="${APP_BASE_URL:-localhost}"
+APP_PORT="${APP_PORT:-22300}"
+DB_CLIENT="${DB_CLIENT:-pg}"
+POSTGRES_PASSWORD="${POSTGRES_PASSWORD:-joplin}"
+POSTGRES_DATABASE="${POSTGRES_DATABASE:-joplin}"
+POSTGRES_USER="${POSTGRES_USER:-joplin}"
+POSTGRES_PORT="${POSTGRES_PORT:-5432}"
+POSTGRES_HOST="${POSTGRES_HOST:-localhost}"
```

Comment:
This is great and obviates: https://github.com/hi-ko/joplin-server-ubuntu-install/pull/2
Or better said, incorporates it. We just need some of hi-ko's attention to merger.

Reply:
I know I saw it too late that we did the same ;-)